packages/sdk-js/package.json

```diff
@@ -30,15 +30,8 @@
   "author": "Rijk van Zanten <rijkvanzanten@me.com>",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.19.2"
-  },
-  "devDependencies": {
-    "chai": "^4.2.0",
-    "mocha": "^8.2.0",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.34.1",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.0.3"
```

Comment:
@rijkvanzanten those dev dependencies are still needed, aren't they?

Reply:
They're hoisted up to the monorepo root 👍🏻

Reply:
I thought `npm run` only uses the local _node_modules_ folder to look for binaries (like rollup and rimraf) to execute 🤔

Reply:
That's correct! In a Lerna based monorepo though, we run all the commands from the root level (eg `npm run build --scope @directus/sdk-js`) which means all the dev-dependencies can be centralized there. That also means that we de-dupe a lot of the dev dependencies, and ensure we're using the same versions across the ecosystem. See https://github.com/lerna/lerna/blob/main/doc/hoist.md for more information

Reply:
Thanks for the explanation. I was using `npm run build` from the local package's directory, Good to know that there is a better way to do that.

> See https://github.com/lerna/lerna/blob/main/doc/hoist.md for more information

Doesn't that only apply to using `lerna bootstrap`, which the directus monorepo currently does not do?

Regardless of that, I just wanted to make sure the change was intentional, as it seemed like this might have snuck in with the merge commit and as some of the packages inside the monorepo do still have non-hoisted dev dependencies 😉

Reply:
> Doesn't that only apply to using lerna bootstrap, which the directus monorepo currently does not do?

If you hoist all the dev dependencies in the package.json of the root package, a single `npm install` will install everything recursively, as we specify the sub-packages as file pointers:

https://github.com/directus/directus/blob/d08c91389ad76cf2b9040a55fec2d2e7c18e7653/package.json#L186-L193

Pretty cool—but very confusing—stuff 🙂 

> Regardless of that, I just wanted to make sure the change was intentional, as it seemed like this might have snuck in with the merge commit [...]

Much appreciated 👏 I can see how this feels a little unintuitive 👍 

> and as some of the packages inside the monorepo do still have non-hoisted dev dependencies 😉

Yeah good catch, there was a note in lerna's docs about keeping "binaries" under the devDependencies of the individual packages, but if you run it from the root with the `--scope` thing this isn't needed. We should remove those  too 👍

Reply:
> If you hoist all the dev dependencies in the package.json of the root package, a single `npm install` will install everything recursively, as we specify the sub-packages as file pointers

Oh, I see! I always wondered why a single `npm install` is enough. Also, I thought _hoisting_ would only refer to automatically "pulling" the dependencies up to the root. Pretty cool indeed 🙂

> there was a note in lerna's docs about keeping "binaries" under the devDependencies of the individual packages

That is actually exactly why I raised the issue 😅